src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -19,6 +20,14 @@ internal abstract partial class BaseAddressOfExpression : Operation, IAddressOfE
         /// Addressed reference.
         /// </summary>
         public abstract IOperation Reference { get; }
+        public override IEnumerable<IOperation> Children
+        {
+            get
+            {
+                yield return Reference;
```

Comment:
Nit: All of these implementations appear to have an empty line at the end of the method.